### PR TITLE
[Draft] : partial implemented sheet page

### PR DESCRIPTION
**Pull request by ankitkumar0622**

### CODE REVIEW

File: src/constant/routes.ts [hunk 1]
```diff
@@ -18,6 +18,10 @@ const routes = {
   },
   interviewPrep: '/interview-prep',
   interviewPrepExplore: '/interview-prep/explore',
+  allSheets: {
+    javaScriptInterviewSheet: '/interview-prep/javascript-interview-sheet',
+    javaScriptBasicsInterviewSheet: '/interview-prep/javascript-basics-interview-sheet',
```
Comment (imsks) [inline src/constant/routes.ts]: Currently do it for JS only. Remove second route and also from other places. Use Search in VS code and remove

**Review by imsks — CHANGES_REQUESTED** [review]

File: src/constant/routes.ts (hunk 1 above)
Comment (BunnyBharath91) [inline src/constant/routes.ts]: remove:  ` javaScriptBasicsInterviewSheet: '/interview-prep/javascript-basics-interview-sheet',`

File: src/constant/pages/seo.ts [hunk 2]
```diff
@@ -80,6 +80,20 @@ const getSEOMeta = (basePath: any): GetSEOMetaResponseType => {
...
+      url: routes.allSheets.javaScriptInterviewSheet,
+      ...commonMeta,
+    },
+    [`${routes.allSheets.javaScriptBasicsInterviewSheet}`]: {
+      title: `JavaScript Basics Interview Sheet | The Boring Education`,
+      siteName: `JavaScript Basics Interview Sheet`,
+      description: `Learn JavaScript basics for interview preparation.`,
+      url: routes.allSheets.javaScriptBasicsInterviewSheet,
+      ...commonMeta,
+    },
```
Comment (BunnyBharath91) [inline src/constant/pages/seo.ts]: remove this

File: src/constant/pages/seo.ts [hunk 3]
```diff
@@ -80,6 +80,20 @@ const getSEOMeta = (basePath: any): GetSEOMetaResponseType => {
       url: routes.allCourses.basicsOfProgrammingWithJS,
       ...commonMeta,
     },
+    [`${routes.allSheets.javaScriptInterviewSheet}`]: {
+      title: `JavaScript Interview Sheet | The Boring Education`,
+      siteName: `JavaScript Interview Sheet`,
+      description: `Prepare for JavaScript interviews with essential questions.`,
```
Comment (BunnyBharath91) [inline src/constant/pages/seo.ts]: replace it by : 
 ```
title: `${INTERVIEW_PREP_SHEETS[0].title} | The Boring Education`,
      siteName: INTERVIEW_PREP_SHEETS[0].title,
      description: INTERVIEW_PREP_SHEETS[0].content, 
```
by importing INTERVIEW_PREP_SHEETS for clean code.

File: src/utils/functions.ts
Comment (BunnyBharath91) [inline src/utils/functions.ts:1]: In the line 229, replace 'shiksha' with 'interview-prep'. 
 ``href: `/interview-prep/${slug}/?sheetId=${_id}`,``

**Review by BunnyBharath91 — COMMENTED** [review]
Resolve all the commented issues. And you are good to go.

**Review by BunnyBharath91 — COMMENTED** [review]
remove the javascript-basics-interview-sheet object from the _'/constant/pages/index.ts'_ file as we don't need it right now.

**Review by imsks — APPROVED** [review]

**Pull request merged**